opentracing/src/main/java/opentracing/propagation/SplitTextFormatInjector.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface SplitTextFormatInjector extends Injector<Map> {
```

Comment:
@michaelsembwever:

> The user does not care which of the map entries are what unless they are working/implementing off the internal api, hence the prefixes are appropriate.

The entire purpose of the "split" carriers is that the user *does* care which entries are tracer state and which are baggage. This is described here: http://opentracing.io/spec/#required-injectorextractor-formats

(FYI, this is not some imagined use-case: Uber's protocols need to put the (fixed-length) tracer state in one place and the baggage in another when RPCs are made. Also, some organizations would be freaked out about Baggage in general, and by exposing it separately it gives OT users an opportunity to keep trace state propagation but guarantee that Baggage never gets sent over the wire.)